changelogs/current.yaml

```diff
@@ -24,5 +24,9 @@ new_features:
     Change HTTP status to 200 to respect the gRPC protocol. This may cause problems for incorrect gRPC clients expecting the filter
     to preserve HTTP 1.1 responses.  This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.grpc_http1_reverse_bridge_change_http_status`` to false.
+- area: healthcheck
+  change: |
+    Added support to healthcheck with ProxyProtocol in TCP Healthcheck by setting
+    :ref:`health_check_config <envoy_v3_api_field_config.core.v3.HealthCheck.TcpHealthCheck.proxy_protocol_config>`
```

Comment:
This line needs to end with a period to make the linter happy.

Reply:
Thank you for pointing. Fixed https://github.com/envoyproxy/envoy/pull/31691/commits/d25971baac735e4329a8b4bce1e1c0178e939020